dali/python/backend_impl.cc

```diff
@@ -1099,7 +1099,7 @@ PYBIND11_MODULE(backend_impl, m) {
   types_m.add_object("CPU_ONLY_DEVICE_ID", PyLong_FromLong(CPU_ONLY_DEVICE_ID));
 
   // DALIDataType
-  py::enum_<DALIDataType>(types_m, "DALIDataType", "Data type of image")
+  py::enum_<DALIDataType>(types_m, "DALIDataType", "Data type of image.\n<SPHINX_IGNORE>")
```

Comment:
Meh

Reply:
I can probably change the docstring in the documenter of enum by hand, knowing that I need to skip everything starting from the `Members:`, but if we update pybind it can change.

Reply:
Ok, let us leave it as it is.

Reply:
But won't it display this garbage in Python's help or IDEs?

Reply:
I wouldn't call it garbage, no more than what the pybind does itself. We already sprinkle the docs with loads of Sphinx tags a annotations, and the main use case is the HTML page.

Reply:
Still meh, but so be it.